development/sass/modules/_hero.scss

```diff
@@ -0,0 +1,95 @@
+.hero {
+  font-family: medium-content-sans-serif-font, 'Lucida Grande', 'Lucida Sans Unicode', 'Lucida Sans', Geneva, Arial, sans-serif;
+  color: white; 
+  position: relative;
+  box-sizing: border-box;
+
+  /* Container for title and button */
+  div {
+    position: absolute;
+    top: 68px;
+    width: 100%;
+    text-align: center;
+
+    @media (min-width: 1000px) {
+      top: 120px;
+    }
+  }
+
+  /* Background image */
+  img {
+    background: url("/resources/images/banners/pokemon-banner.jpg") no-repeat center 0 fixed; 
```

Comment:
You don't apply a background to an image, apply this to the hero itself. It should actually be applied to a decoration class such as `.background-bikes`.